docs/user-guide.md

```diff
@@ -0,0 +1,856 @@
+* tell PyScript how to run your Python code.
+
+For the browser to use PyScript, simply add a `<script>` tag, whose `src`
+attribute references a CDN url for `pyscript.core`, to your HTML document's
+`<head>`. You may also add a reference to optional PyScript related CSS:
+
+**TODO: USE CORRECT CDN URL**:
+
+```html title="Reference PyScript in your HTML"
+<!doctype html>
+<html>
+    <head>
+        <!-- Recommended meta tags -->
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
+        <!-- optional PyScript CSS -->
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@pyscript/core/dist/core.css">
+        <!-- This script tag bootstraps PyScript -->
+        <script type="module" src="https://cdn.jsdelivr.net/npm/@pyscript/core"></script>
+    </head>
+    <body>
+        <!-- your code goes here... -->
+    </body>
+</html>
+```
+
+There are two ways to tell PyScript how to find your code.
+
+* With a standard HTML `<script>` tag whose `type` attribute is either `py`
+  (for Pyodide) or `mpy` (for MicroPython). **This is currently the
+  recommended way**.
+* Via the bespoke `<py-script>` (Pyodide) and `<mpy-script>` (MicroPython)
+  tags. Historically, this used to be the only way to reference your code.
+
+These should be inserted into the `<body>` of your HTML document.
+
+In both cases either use the `src` attribute to reference a Python
+file containing your code, or inline your code between the opening and closing
+tags. **We recommend you use the `src` attribute method**, but retain the
+ability to include code between tags for convenience.
+
+Here's the `<script>` tag method using the `src` attribute to reference the URL
+for a `main.py` Python file.
+
+```html title="Using the script tag with a source file"
+<script type="mpy" src="main.py"></script>
+```
+
+...and here's the `<py-script>` tag with inline Python code.
+
+```html title="Using the py-script tag with inline code"
+<py-script>
+import sys
+from pyscript import display
+
+
+display(sys.version)
+</py-script>
+```
+
+The browser's tab displaying the website running PyScript is an isolated
+computing sandbox. Define the Python environment in which your code will
+run with [configuration options](#configuration) (discussed later in this
+document).
+
+The `<script>` and `<py-script>` / `<mpy-script>` tags can have the following
+attributes:
+
+* `src` - the content of the tag is ignored and the Python code in the
+  referenced file is evaluated instead. **This is the recommended way to
+  reference your Python code.**
+* `config` - your code will only be evaluated after the referenced
+  configuration has been parsed. Since configuration can be JSON (or a TOML
+  file), `config='{"packages":["numpy"]}'` and `config="./config.json"` or
+  `config="./config.toml"` are all valid.
+* `async` - your Python code is run asynchronously (awaited), rather than in a
+  blocking manner.
```

Comment:
I don't think it's true.
What it does is to allow `top level await`, but the code is still sync.
Yes, I think it's a wrong name :man_shrugging:

Reply:
OK... let's figure out a better / more accurate way to put this next week. It's now on my todo list.